docker/registry/go/gradle.properties

```diff
@@ -0,0 +1,3 @@
+io.deephaven.project.ProjectType=DOCKER_REGISTRY
+deephaven.registry.imageName=golang:1.18.1
```

Comment:
I think we should prefer to track `golang:1`. (`1.18.4` is latest available.) If a golang update `1.19` causes the build to break, I can explicitly track `1.18` at that point in time.

Reply:
Decided in slack that we're going to assume 1.x here.